Formula/go@1.12.rb

```diff
@@ -13,8 +13,6 @@ class GoAT112 < Formula
 
   keg_only :versioned_formula
 
-  depends_on :macos => :yosemite
```

Comment:
Rather change this to `depends_on :macos => :el_capitan`

Reply:
That is right, let me do that.

Reply:
Wait — I got confused. The Go 1.12 release line _does_ still support Yosemite, so don't change the line at all :)

Reply:
yeah, I just realized that too, I guess it only applies to 1.13. 👻

Reply:
I'm sorry! 😳